.github/workflows/branch_sync.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         id: diff
         if: steps.merge.outcome == 'success'
         run: |
-          if [[ "$(git rev-parse HEAD)" == "$(git rev-parse master)" ]]; then
+          if [[ "$(git rev-parse HEAD)" == "$(git rev-parse origin/master)" ]]; then
```

Comment:
Oops, as master is currently checked out this was always true.

(Didn't pick up in my testing as I was using a different branch to master